rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -144,6 +145,14 @@ void GLGSRender::on_init_thread()
 		}
 	}
 
+	if (gl_caps.NV_fragment_shader_barycentric_supported &&
+		gl_caps.vendor_NVIDIA &&
+		g_cfg.video.shader_precision != gpu_preset_level::low)
+	{
+		// NVIDIA's attribute interpolation requires some workarounds
+		backend_config.supports_normalized_barycentrics = false;
+	}
```

Comment:
the logic here seems to differ to the logic in Vulkan

Reply:
It's the same logic, just applied forwards vs backwards.
In vulkan its a forward assignment.
Here its an inverse assignment because we don't have the caps at the point of construction.